### PR TITLE
Education-Translations (Turkish) (Trading Bot Layers Tutorial)

### PR DESCRIPTION
**Pull request by prepheadrus**
Docs-Nodes\T\Trading\Trading-Episode-Counters\trading-episode-counters.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-010-tutorial-step-1-2-3-counting.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-011-tutorial-step-trading-bot-layers-episode-counters-01.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-012-tutorial-step-how-about-the-rest.json
Docs-Nodes\E\Entry\Entry-Target-Rate\entry-target-rate.json
Docs-Nodes\E\Exit\Exit-Target-Rate\exit-target-rate.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-015-tutorial-step-elephant-what-elephant.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-016-tutorial-step-making-sense-of-objects.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-017-tutorial-step-what-about-the-position.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-018-tutorial-step-the-beauty-of-the-triangle.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-019-tutorial-step-why-3-position-layers.json
Docs-Nodes\E\Episode\Episode-Quoted-Asset\episode-quoted-asset.json
Docs-Nodes\M\Market\Market\market.json
Docs-Nodes\O\Order\Order-Base-Asset\order-base-asset.json
Docs-Nodes\O\Order\Order-Quoted-Asset\order-quoted-asset.json
Docs-Nodes\O\Orders\Orders\orders.json
Docs-Nodes\P\Position\Position-Base-Asset\position-base-asset.json
Docs-Nodes\P\Position\Position-Quoted-Asset\position-quoted-asset.json
Docs-Nodes\S\Stage\Stage-Base-Asset\stage-base-asset.json
Docs-Nodes\S\Stage\Stage-Quoted-Asset\stage-quoted-asset.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-020-tutorial-step-remember-flexibility.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-021-tutorial-step-phased-management.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-022-tutorial-step-orders-in-order.json
Docs-Nodes\P\Periods\Periods\periods.json
Docs-Nodes\P\Positions\Positions\positions.json
Docs-Nodes\S\Strategies\Strategies\strategies.json
Docs-Nodes\T\Trading\Trading-Current\trading-current.json
Docs-Nodes\T\Trading\Trading-Episode\trading-episode.json
Docs-Nodes\M\Manage\Manage-Stage\manage-stage.json
Docs-Topics\T\Trading\Trading-Systems\trading-systems-007-the-manage-stage.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-023-tutorial-step-stages-time.json
Docs-Nodes\T\Trading\Trading-Engine\trading-engine.json
Docs-Nodes\C\Close\Close-Stage\close-stage.json
Docs-Nodes\O\Open\Open-Stage\open-stage.json
Docs-Nodes\T\Trigger\Trigger-Stage\trigger-stage.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-024-tutorial-step-episode-x.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-025-tutorial-step-understanding-objects-data.json
Docs-Nodes\T\Trading\Trading-System\trading-system.json
Docs-Nodes\F\Formula\Formula\formula.json
Docs-Nodes\I\Initial\Initial-Targets\initial-targets.json
Docs-Nodes\S\Strategy\Strategy\strategy.json
Docs-Nodes\T\Trading\Trading-Strategy\trading-strategy.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-026-tutorial-step-daily-files-vs.-market-files.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-027-tutorial-step-bring-me-back-to-earth.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-028-tutorial-topic-trading-system-layers.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-029-tutorial-step-trading-system-errors.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-030-tutorial-step-trading-system-highlights.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-033-tutorial-step-trading-system-values.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-031-tutorial-step-trading-bot-layers-trading-system-highlights.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-032-tutorial-step-quick-refresher-on-stages.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-034-tutorial-step-the-rest-of-the-crew.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-035-tutorial-step-time-to-go-out-and-play.json

### CODE REVIEW

**Review by gitforkgit — DISMISSED** [review]
Everything look fine to me. I couldn`t see any uncommonities. @harrellbm From my perspective this Pull request is able to merge. :)

**Comment by harrellbm** [comment]
@prepheadrus @gitforkgit With the PRs that got merged before this one there are now a few conflicts.  You should be able to use the forum post about conflicts to fixe them and then we can get this merged!

**Comment by gitforkgit** [comment]
@prepheadrus This should be a quick fix. Just click on Resolve conflicts and follow the short tutorial from @harrellbm: https://forum.superalgos.org/t/how-to-resolve-a-conflicting-translation/103?u=harrellbm

**Comment by prepheadrus** [comment]
> @prepheadrus This should be a quick fix. Just click on Resolve conflicts and follow the short tutorial from @harrellbm: https://forum.superalgos.org/t/how-to-resolve-a-conflicting-translation/103?u=harrellbm

hey @gitforkgit , i saw it and fixed it..thanks

**Comment by gitforkgit** [comment]
@prepheadrus Awesome, i'll look at the pull request again and mark it as mergable if everything fits :+1:

**Comment by prepheadrus** [comment]
> 👍

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:47]: The bracket before the spanish translation is not in line with the closing bracket from the Turkey one.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json:135]: Brackets not in line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json:83]: Brackets not in line

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json:160]: Brackets not in line.

**Review by gitforkgit — COMMENTED** [review]
I think it are only those very small and easy fixes. After you corrected them, I will go trough all files and then approve if nothing appears to be wrong to me.
Only a small step left and then we can merge :+1:

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json]: fix it.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json]: fix it

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json]: fix it

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json]: fix it

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json
Comment (gitforkgit) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json]: :+1: I will review again

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:149]: Those brackets need to be more left, maybe you can orient on already merged translations or files you also commited a bit more down. This occurs at some parts in this file.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:173]: Same here.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:251]: same here

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json:109]: Same here.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:72]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:127]: Brackets must be more left.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:153]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:211]: Brackets must be more left.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:215]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:255]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:90]: Brackets must be more left.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:177]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:246]: Blank line.

**Review by gitforkgit — CHANGES_REQUESTED** [review]
Only few small adjustements left. Beside the Two Files all other 54 Files look great. Everywhere where i commented Blank line, you don`t need but can remove them. If you don`t see the brackets aligned wrong, let me know. Only a small step left before merging.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json]: I did not fully understand. can you be a little more clear?

**Comment by gitforkgit** [comment]
@harrellbm can you have a look?

**Review by harrellbm — APPROVED** [review]

**Pull request merged**
